CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.16.0
+
+* Components and resources no longer need to be `Send + Sync` if parallel feature is disabled ([#673])
```

Comment:
**TODO:** Likely should be component *storages*. `Component`s still have to be `Send + Sync`.